toolflow/vivado/platform/xupvvh/xupvvh.tcl

```diff
@@ -229,7 +229,7 @@ namespace eval platform {
     insert_regslice "dma_host" true "/memory/M_HOST" "/host/S_HOST" "/clocks_and_resets/host_clk" "/clocks_and_resets/host_interconnect_aresetn" ""
     insert_regslice "host_arch" true "/host/M_ARCH" "/arch/S_ARCH" "/clocks_and_resets/design_clk" "/clocks_and_resets/design_interconnect_aresetn" ""
 
-    if {[tapasco::is_feature_enabled "SFPPLUS"]} {
+    if {[get_bd_intf_pins "/network/S_NETWORK"] ne ""} {
```

Comment:
What is the reason behind this change?

Reply:
Not sure, this is part of your commit (4b617b5337d2870c536dba8f6c967811209dd567). Should we undo it?

Reply:
Ah, you're right.
Now I also remember the reason: In the 100G mode the interface `/network/S_NETWORK` does not exist and thus the old version would fail. Is this still the case or does the 100G mode create this interface now?

Reply:
`/network/S_NETWORK` does not exist for 100G, as we do not use the AXI Lite control interface for CMAC (we use ports `ctl_rx_enable` etc instead, which would be removed if we enable the AXI interface)